package.json

```diff
@@ -61,20 +57,20 @@
     "lodash": "~3.10.1",
     "main-bower-files": "~2.9.0",
     "ng-annotate-loader": "0.0.10",
+    "ng-csv": "^0.3.6",
     "phantomjs": "~1.9.18",
     "phantomjs-polyfill": "0.0.1",
     "uglify-save-license": "~0.4.1",
+    "underscore": "^1.8.3",
```

Comment:
Its not a change I made. I added only isparta, needed it for testing.

Will look into underscore and see if its needed